src/tlo/methods/bed_days.py

```diff
@@ -145,6 +145,40 @@ def initialise_beddays_tracker(self, model_to_data_popsize_ratio=1.0):
             assert not df.isna().any().any()
             self.bed_tracker[bed_type] = df
 
+    def switch_beddays_availability(
+        self,
+        new_availability: Literal["all", "none", "default"],
+        effective_on_and_from: Date,
```

Comment:
Happy to make a public/private method like is suggested above, though I'd quite like to avoid spreading the `self.sim.modules...` cross-class-coupling into the bed-days class if possible. `switch_beddays_availability` is [already called by classes](https://github.com/UCL/TLOmodel/blob/6b86f76d44335d1430c31e153b5f9e5080f47de0/src/tlo/methods/healthsystem.py#L2779) that have this cross-class-coupling anyway, so it might as well stay isolated to there?